src/test/kotlin/no/nav/familie/ef/sak/beregning/BeregningBarnetilsynServiceTest.kt

```diff
@@ -41,116 +43,130 @@ class BeregningBarnetilsynServiceTest {
     val februar2000 = YearMonth.of(2000, 2)
     val mars2000 = YearMonth.of(2000, 3)
 
+    @Nested
+    inner class BeregningBarnetilsynValidering {
 
-    @Test
-    internal fun `Skal kaste feil når vi sender inn ufornuftige beløp `() {
-        val utgiftsperiode = listeMedEnUtgiftsperiode()
-        val utgiftsperiodeMedHøytBeløp = listeMedEnUtgiftsperiode(beløp = 50000)
-        val utgiftsperiodeMedNegativtBeløp = listeMedEnUtgiftsperiode(beløp = -1)
-        val periodeMedHøytBeløp = listeMedEnPeriodeMedBeløp(beløp = 50000)
-        val periodeMedNegativtBeløp = listeMedEnPeriodeMedBeløp(beløp = -1)
-
-        val negativUtgiftsfeil = assertThrows<ApiFeil> {
-            service.beregnYtelseBarnetilsyn(utgiftsperioder = utgiftsperiodeMedNegativtBeløp,
-                                            kontantstøttePerioder = listOf(),
-                                            tilleggsstønadsperioder = listOf())
+        @Test
+        internal fun `Skal kaste feil når vi sender inn kontantstøtteperiode-fradrag før kontantstøtte ble innført`() {
+            val utgiftsperiode = listeMedEnUtgiftsperiode()
+            val perioderMedTidligDato = listeMedEnPeriodeMedBeløp(fra = YearMonth.of(2020, Month.FEBRUARY))
+            val kontantstøtteperiodeStarterForTidlig = assertThrows<ApiFeil> {
+                service.beregnYtelseBarnetilsyn(utgiftsperioder = utgiftsperiode,
+                                                kontantstøttePerioder = perioderMedTidligDato,
+                                                tilleggsstønadsperioder = listOf())
+            }
+            assertThat(kontantstøtteperiodeStarterForTidlig.message).isEqualTo("Fradrag for innvilget kontantstøtte trår i kraft: 2020-03")
         }
-        assertThat(negativUtgiftsfeil.message).isEqualTo("Utgifter kan ikke være mindre enn 0")
 
```

Comment:
La til @Nested for å dele opp alle testene litt. Ble litt rotete - ingen endringer i testene under her.